conda_index/index/__init__.py

```diff
@@ -858,7 +713,7 @@ def __init__(
         self.thread_executor = (
             DummyExecutor()
             if (debug or sys.version_info.major == 2 or threads == 1)
-            else ProcessPoolExecutor(threads)
+            else ProcessPoolExecutor(threads, initializer=logging_config)
```

Comment:
The final missing piece. I was surprised fork() didn't share all log configs in our subprocess. Now we err towards too much logging, we will need to pass the level or "verbose or not" to the subprocess.